apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.sql

```diff
@@ -8,9 +8,9 @@ WHERE
   AND deployment_id = $deployment_id
   AND context_id = $context_id;
 
--- BLOCK select_lti13_course_instance_institution
+-- BLOCK select_lti13_institution_course_instance
```

Comment:
Consider moving the `ci.id = $course_instance_id` into the `WHERE` clause; it took me a while to find that this was actually filtering down to a particular course instance.

Reply:
Addressed in 69f0ba1